### PR TITLE
[REGEDIT] *.rc Improve accels and harmonize a bit, especially de-DE

### PR DESCRIPTION
**Pull request by JoachimHenze**
* all languages: Strip unintended accelerators for Cancel-buttons
* many languages: had accel collision between "&File"-menu and "&Favorites" menu. It must be "F&avrites" for those!
* de-DE: "&Wertname:" -> "&Name:" was done because this is what Windows does and to fix an accel collision between "&Wert" and "&Wertname"
* de-DE: use the same name for String, DWORD, MULTISTRING, EXPANDABLESTRING in all 3 dialogs (each of the 3 dlgs used different names before), which also fixes an accel collision between "&Schlüssel" and "&String"
* de-DE: structurally use "Sc&hlüssel", instead of mixed things
* de-DE: "&Trennen" -> "&Teilen" is what Windows does
* de-DE "&Ändern" -> "Än&dern" is what Windows does, and avoids accel at a special character
* de-DE "&Über" -> "Inf&o" is what Windows does, and avoids accel at a special character
* de-DE "S&chlüsselnamen kopieren" -> "Schlüsselnamen &kopieren" is to match another already existing "Schlüsselnamen &kopieren" in the file
* de-DE "A&lles auswählen" -> "Alles &markieren" is what Windows does
* all languages: fix 26x wrong capitalization for "Ms Shell Dlg" -> "MS Shell Dlg"
* use the correct fonts for some asian languages (font-size9) this was for 5 new dlgs from 0.4.15-dev'ing
* fix many accelerator collisions for many languages, or leave FIXMEs for them where I found it to be better for the native speakers to act on their own in the future

JIRA issue: none

### CODE REVIEW

**Comment by HBelusca** [comment]
Tell us when the PR is ready for review ;)

**Comment by JoachimHenze** [comment]
> Tell us when the PR is ready for review ;)

You can do suggestions already @hbelusca. If you have a Windows-double-checked accelerator proposal for one of the FIXMEs, you can tell.

**Comment by JoachimHenze** [comment]
Officially **ReadyForReview** now.

File: base/applications/regedit/lang/ru-RU.rc [hunk 1]
```diff
@@ -83,9 +83,9 @@ BEGIN
     POPUP "&Файл"
     BEGIN
         MENUITEM "&Импортировать из файла...", ID_REGISTRY_IMPORTREGISTRYFILE
-        MENUITEM "&Экспортировать в файл...", ID_REGISTRY_EXPORTREGISTRYFILE
+        MENUITEM "&Экспортировать в файл...", ID_REGISTRY_EXPORTREGISTRYFILE //FIXME: accelerator collision &Э
         MENUITEM SEPARATOR
-        MENUITEM "&Загрузить куст...", ID_REGISTRY_LOADHIVE, GRAYED
+        MENUITEM "&Загрузить куст...", ID_REGISTRY_LOADHIVE, GRAYED //FIXME: accelerator collision &Э. I decided to use "З&агрузить куст..." to fix the collision in the older releases. But I don't have a russian Windows.
```
Comment (binarymaster) [inline base/applications/regedit/lang/ru-RU.rc:88]: Man... there's no collision, because "З" and "Э" are two distinct cyrillic letters.

Comment (binarymaster) [reply base/applications/regedit/lang/ru-RU.rc:88]: image

Comment (JoachimHenze) [reply base/applications/regedit/lang/ru-RU.rc:88]: Good catch. NPP6.7.9.2 matches them both when searching for "&Э".

**Pull request merged**